File: LeetCode/1768_Merge_Strings_Alternately/solution.js
/**
 * @param {string} word1
 * @param {string} word2
 * @return {string}
 */
var mergeAlternately = function(word1, word2) {
    let i=word1.length>word2.length?word1.length:word2.length;
    let res=""
    for(let j=0; j<i;j++){
        if(word1[j]!=undefined)res+=word1[j]
        if(word2[j]!=undefined)res+=word2[j]
    }
    return res;
};

File: LeetCode/1768_Merge_Strings_Alternately/solution.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int i = word1.length() > word2.length() ? word1.length() : word2.length();
        string res = "";
        for (int j = 0; j < i; j++) {
            if(j<word1.length()){
                res += word1[j];
            }
            
            if(j<word2.length()){
                res += word2[j];
            }
            
        }
        return res;
    }
};